src/lib/tecs/TECS.hpp

```diff
@@ -203,8 +219,15 @@ class TECS
 
 private:
 
+	// [m/s] bound on expected (safe) true airspeed tracking errors, including TAS = TAS_min - TAS_ERROR_BOUND
+	static constexpr float TAS_ERROR_BOUND = 2.0f;
+
+	// [m/s] true airspeed soft boundary region below the accepted TAS error region (below TAS_min - TAS_ERROR_BOUND)
+	// underspeed mitigation measures are ramped in from zero to full within this region
+	static constexpr float TAS_UNDERSPEED_SOFT_BOUND = 1.5f;
```

Comment:
put to hardcoded 15%. this can be changed to param later on if people want to expose it. [e13ebc7](https://github.com/PX4/PX4-Autopilot/pull/19869/commits/e13ebc74cfff4b10e599b9fe325891165d4747fc)